robotoff/app/api.py

```diff
@@ -1177,6 +1178,41 @@ def on_get(self, req: falcon.Request, resp: falcon.Response):
         resp.media = response
 
 
+class LogoAnnotationCollection:
+    def on_get(self, req: falcon.Request, resp: falcon.Response):
+        response: JSONType = {}
+        barcode: Optional[str] = req.get_param("barcode")
+        keep_types: Optional[List[str]] = req.get_param_as_list("types", required=False)
```

Comment:
I am naming the parameter as `types`. But in the test case it only works when I pass it as `keep_types`